org.gecko.vaadin.whiteboard.demo/launch_whiteboard_demo_osgi.bndrun

```diff
@@ -41,7 +41,7 @@
 	org.apache.commons.io;version='[2.5.0,2.5.1)',\
 	org.apache.commons.logging;version='[1.2.0,1.2.1)',\
 	org.apache.felix.configadmin;version='[1.9.18,1.9.19)',\
-	org.apache.felix.configurator;version='[1.0.8,1.0.9)',\
+	org.apache.felix.configurator;version='[1.0.12,1.0.13)',\
```

Comment:
I believe this line is not needed. Please verify if the build still works without this line.

Reply:
This was changed because of build issues. It would not have been changed otherwise.